src/main/java/org/openmrs/addonindex/domain/artifactory/ArtifactoryFolderInfo.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.addonindex.domain.artifactory;
+
+import java.util.List;
+
+/**
+ * Represents the list of OpenMRS core versions we want to index.
```

Comment:
I would say:
* represents the response from (link to the REST call)
* we use this to determine the list of OpenMRS core versions